.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "server/lib/weboob/cozyweboob"]
+	path = server/lib/weboob/cozyweboob
+	url = https://github.com/Phyks/cozyweboob
+[submodule "server/konnectors/weboob/cozyweboob"]
+	path = server/konnectors/weboob/cozyweboob
+	url = https://github.com/Phyks/cozyweboob
```

Comment:
Indeed. This is a Python code, I should check how I can reliabily distribute it with NPM.